xbmc/PlayListPlayer.cpp

```diff
@@ -892,6 +894,12 @@ void PLAYLIST::CPlayListPlayer::OnApplicationMessage(KODI::MESSAGING::ThreadMess
         if (list->Size() == 1 && !(*list)[0]->IsPlayList())
         {
           CFileItemPtr item = (*list)[0];
+          // if the item is a plugin we need to resolve the URL since the infotags are only
+          // available in case the item was played before (stored on the database)
+          for (int i=0; URIUtils::IsPlugin(item->GetDynPath()) && i<5; ++i)
```

Comment:
Where does the magic 5 come from?

And please follow our coding guidelines. blanks before and after "=" and "<", please.

Reply:
From nowhere :) other components in kodi do the same. It just means you'll try to resolve the url recursively for a maximum of 5 times. Plugin:// paths can resolve to plugins paths so this avoids entering in some kind of deadlock. Any suggestions?

Regarding style issues, I'll do.